[2022-03-15 10:36:15,053] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-15T10:36:01.546722+00:00 [queued]>
[2022-03-15 10:36:15,065] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-15T10:36:01.546722+00:00 [queued]>
[2022-03-15 10:36:15,065] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 10:36:15,065] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 10:36:15,065] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 10:36:15,083] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): daily_analysis> on 2022-03-15 10:36:01.546722+00:00
[2022-03-15 10:36:15,087] {standard_task_runner.py:52} INFO - Started process 1283 to run task
[2022-03-15 10:36:15,091] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '2_daily_transformation_analysis', 'daily_analysis', 'manual__2022-03-15T10:36:01.546722+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/transform_and_analysis.py', '--cfg-path', '/tmp/tmp9is9ahpm', '--error-file', '/tmp/tmpsn63k2fy']
[2022-03-15 10:36:15,091] {standard_task_runner.py:77} INFO - Job 65: Subtask daily_analysis
[2022-03-15 10:36:15,150] {logging_mixin.py:109} INFO - Running <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-15T10:36:01.546722+00:00 [running]> on host dd3c58b082a3
[2022-03-15 10:36:15,248] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=2_daily_transformation_analysis
AIRFLOW_CTX_TASK_ID=daily_analysis
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T10:36:01.546722+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-15T10:36:01.546722+00:00
[2022-03-15 10:36:15,249] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 10:36:15,249] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /dbt && dbt run --models analysis --profiles-dir .']
[2022-03-15 10:36:15,259] {subprocess.py:85} INFO - Output:
[2022-03-15 10:36:17,894] {subprocess.py:89} INFO - Running with dbt=0.20.0
[2022-03-15 10:36:20,345] {subprocess.py:89} INFO - Found 5 models, 0 tests, 0 snapshots, 0 analyses, 338 macros, 0 operations, 3 seed files, 0 sources, 0 exposures
[2022-03-15 10:36:20,346] {subprocess.py:89} INFO - 
[2022-03-15 10:36:22,297] {subprocess.py:89} INFO - 10:36:22 | Concurrency: 200 threads (target='dev')
[2022-03-15 10:36:22,297] {subprocess.py:89} INFO - 10:36:22 |
[2022-03-15 10:36:22,317] {subprocess.py:89} INFO - 10:36:22 | 1 of 2 START view model analysis.hotel_count_by_day.................. [RUN]
[2022-03-15 10:36:22,318] {subprocess.py:89} INFO - 10:36:22 | 2 of 2 START view model analysis.thirty_day_avg_cost................. [RUN]
[2022-03-15 10:36:23,446] {subprocess.py:89} INFO - 10:36:23 | 2 of 2 OK created view model analysis.thirty_day_avg_cost............ [[32mSUCCESS 1[0m in 1.13s]
[2022-03-15 10:36:23,462] {subprocess.py:89} INFO - 10:36:23 | 1 of 2 OK created view model analysis.hotel_count_by_day............. [[32mSUCCESS 1[0m in 1.14s]
[2022-03-15 10:36:23,989] {subprocess.py:89} INFO - 10:36:23 |
[2022-03-15 10:36:23,989] {subprocess.py:89} INFO - 10:36:23 | Finished running 2 view models in 3.64s.
[2022-03-15 10:36:23,998] {subprocess.py:89} INFO - 
[2022-03-15 10:36:23,998] {subprocess.py:89} INFO - [32mCompleted successfully[0m
[2022-03-15 10:36:23,998] {subprocess.py:89} INFO - 
[2022-03-15 10:36:23,999] {subprocess.py:89} INFO - Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2022-03-15 10:36:25,221] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 10:36:25,256] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=2_daily_transformation_analysis, task_id=daily_analysis, execution_date=20220315T103601, start_date=20220315T103615, end_date=20220315T103625
[2022-03-15 10:36:25,321] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 10:36:25,354] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
